### ISSUE TITLE
Enable configuring the Conda / pip environment to run flake8 in

### ISSUE THREAD

**Issue by PeterFogh** [mlshapiro/jupyterlab-flake8#10]
Hi,
First of all thanks for this very useful extension.

It is possible to support configuration of what python environment flake8 is executed in.

After a quit view in your code, a suspect the activation of an environment can be implemented in the following line in https://github.com/mlshapiro/jupyterlab-flake8/blob/5e123881f207112bfe81fc918819b80570f59ab9/src/index.ts#L149
which can be changed to something like:
```
    this.term = new Terminal({initialCommand: 'echo "Opening flake8 terminal" && source activate [environment_name]'});
```
And one must remember that `source` is used on Unix and not on Windows.

**Comment by mlshapiro** [comment]
Hi @PeterFogh thanks for the suggestion. This does make sense - it is always frustrating to have anaconda pull alternate binary from a different environment.  This may actually solve #7 too.  I'll take a look as soon as I can. Please feel free to make a PR if you feel motivated!

**Comment by mlshapiro** [comment]
implemented in 0.3.0 - see *Advanced Settings* for  `conda_env` setting.
